### DISCUSSION TITLE
3rd party antivirus

### DISCUSSION THREAD

**Discussion by chaselmoodye5** [HotCakeX/Harden-Windows-Security#192] [Q&A]
Hey guys, is it possibly to modify the script to run on a machine that has to have a 3rd part antivirus in place? For example if we would like to maintain SentinelOne endpoint protection. Can I still get this to work? I know I would have to remove some of the script related to defender.

**Comment by HotCakeX** [comment]
Hi,
I'm not sure how a 3rd party EDR would work with the implemented security measures since all of them assume Microsoft Defender is in charge, although MDE EDR works perfectly with the security measures.

So I'd say just skip the Defender category entirely. To be on the safe side skip ASR and possibly Miscellaneous categories too.

**Comment by bigjezza** [comment]
Is it possible to skip the Defender requirement check at our own risk? 

> PS C:\Windows\System32> Protect-WindowsSecurity -Categories MicrosoftSecurityBaselines,DeviceGuard
Protect-WindowsSecurity: Exception calling "RequirementsCheck" with "0" argument(s): "Microsoft Defender Anti Malware service is not enabled, please enable it and then try again."
